projects/nexus-dashboard/day-4/abhishekgoswami/src/styles/index.css

```
@@ -0,0 +1,118 @@
+  --radius:          0.75rem;
+  --transition:      0.25s ease;
+  --shadow:          0 8px 32px rgba(0, 0, 0, 0.5);
+}
+
+/* ── Base ────────────────────────────────────────── */
+*, *::before, *::after { box-sizing: border-box; margin: 0; padding: 0; }
+html { scroll-behavior: smooth; }
+body {
+  background-color: var(--color-bg);
+  color: var(--color-text);
+  font-family: 'Inter', system-ui, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)